config/locales/en.yml

```
@@ -69,3 +69,12 @@ en:
   page_titles:
     error_prefix: "Error: "
     subjects_filter: "Select the subjects you want to teach"
+  service_name:
+    get_into_teaching: Get into teaching
+  get_into_teaching:
+    # TODO: Update to 0800 389 2500 when GIT leaves beta
+    tel: 0800 389 2501
+    opening_times: Monday to Friday, 8.30am to 5pm (except public&nbsp;holidays)
```

Unicode might be a better fit here. Means you don't need the HTML entity and can remove the calls to `.html_safe` in the template.

```suggestion
    opening_times: "Monday to Friday, 8.30am to 5pm (except public\u00A0holidays)"
```

---

Awesome! This is much better, thanks @malcolmbaig 😄